### PR TITLE
Realistic gps

### PR DESCRIPTION
**Pull request by skimslozo**
**Input especially appreciated on:**

- Upper bound on eph and epv values in GpsSimpleParams.h (standard deviations of horizontal and vertical position errors, respectively)
- An alternative method on how to cut-off noise below a certain velocty thershold to avioid GPS read-outs jumping around during standstill or perhaps a smarter method to make it velocity dependent

### CODE REVIEW

**Comment by skimslozo** [comment]
> Nice work! Before I review the code in depth or get to your questions, could you please add some externally-facing documentation about how GPS works so that teams know how to use it in their SE algo's?

Working on that as we speak. :)

File: AirSim/AirLib/include/common/CommonStructs.hpp [hunk 1]
```diff
@@ -122,10 +122,6 @@ struct PoseWithCovariance {
     {}
 };
 
-struct PowerSupply {
-    vector<real_T> voltage, current;
-};
-
```
Comment (skimslozo) [inline AirSim/AirLib/include/common/CommonStructs.hpp:128]: This definitelty should be there, whoops

File: AirSim/AirLib/include/sensors/gps/GpsSimple.hpp [hunk 2]
```diff
@@ -65,24 +68,55 @@ class GpsSimple : public GpsBase {
...
+        auto seed = (unsigned int)std::chrono::system_clock::now().time_since_epoch().count();
+        std::default_random_engine generator(seed);
+        return distribution(generator);
+    }
+    //Generate noisy geo_point for moving vehicle
+    msr::airlib::GeoPoint generateErrors(msr::airlib::GeoPoint geo_point, real_T eph, real_T epv)
+    {
+        HomeGeoPoint geo_point_in = HomeGeoPoint(geo_point);
+        float var_h = pow(eph, 2)/18;
+        float var_v = pow(epv, 2)/9;
```
Comment (SijmenHuizenga) [inline AirSim/AirLib/include/sensors/gps/GpsSimple.hpp:84]: what do these 18 and 9 represent?

Comment (skimslozo) [reply AirSim/AirLib/include/sensors/gps/GpsSimple.hpp:84]: Essentially they are a result of following the [3-sigma rule](https://en.wikipedia.org/wiki/68%E2%80%9395%E2%80%9399.7_rule) for Gaussian distribution so we are sure that the "random" noise we generate are within bounds.

**Comment by SijmenHuizenga** [comment]
I changed the params to what whe discussed, copied the docs from the sensor suite docs document into the docs folder and changed it to reflect the settings we chose. @davidoort @skimslozo can you check these additions?

**Pull request merged**